.github/workflows/workflow.yaml

```diff
@@ -206,12 +206,12 @@ jobs:
         if: inputs.push
         name: Push tags
         id: push
-        env:
-          IMAGE: ${{ inputs.name }}:${{ steps.setup.outputs.unique-id }}
-          TAGS: ${{ join(fromJSON(steps.meta.outputs.json).tags, ' ') }}
         run: |
           echo "### Tags" >> $GITHUB_STEP_SUMMARY
 
+          export IMAGE=${{ inputs.name }}:${{ steps.setup.outputs.unique-id }}
+          export TAGS=${{ join(fromJSON(steps.meta.outputs.json).tags, ' ') }}
```

Comment:
Bruk single-quotes rundt verdiene til variablene for å forhindre word splitting eller ytterligere interpolasjon i bash.

Reply:
Ellers en flott løsning jeg har slitt med å finne løsning på :)

Reply:
Sånn?